ui/com/composer/index.jsx

```diff
@@ -55,20 +56,19 @@ export default class Composer extends React.Component {
     if (this.props.thread) {
       // thread categorization
       this.isThreadPublic = isThreadPublic(this.props.thread)
-      this.threadChannel = (this.props.thread.value.content.channel || '').trim()
+        this.threadChannel = (this.props.thread.value.content.channel || '').trim()
```

Comment:
There are some wonky tab situations, like this here. Some of it is alignment, which I'm totally cool with, but can you clean up the stuff like this? Tabs should be at 2 spaces.